tensorflow/compiler/jit/ops/xla_ops.cc

```diff
@@ -95,4 +95,22 @@ Executes a TensorFlow function previously compiled into a LocalExecutable by an
 _XlaCompile op.
 )");
 
+REGISTER_OP("_XlaMerge")
+    .Input("partitioned_call: T")
+    .Input("xla_run: T")
+    .Output("output: T")
+    .Attr("T: type")
+    .SetShapeFn([](InferenceContext* c) {
+      c->set_output(0, c->input(0));
+      return Status::OK();
+    })
+    .Doc(R"(XLA Merge Op. For use by the XLA JIT only.
+
+Merges the outputs from the PartitionedCall node and the _XlaRun node.
+Unlike the TensorFlow merge op, _XlaMerge supports merging inputs of all types.
```

Comment:
This sounds a bit misleading.  Even `Merge` supports all data types; the difference is that Merge places int32 types on the host whereas `_XlaMerge` places them on the device.

Reply:
agreed. will clarify